android-studio-sync-base.conf

```diff
@@ -0,0 +1,67 @@
+# Performance scenario for testing Android Studio sync performance.
+#
+# Usage: (See also README.md)
+#
+# It requires Android Studio Bumblebee (2021.1.1) or greater
+#
+# 1. Create a project-specific scenario file in the parent directory.
+# 2. Include this file by adding the following in the project-specific scenario file:
+#
+#     include required(file("gradle-benchmark-base/android-studio-sync-base.conf"))
```

Comment:
Can you include both incremental-base and android-studio-sync base?

Reply:
I tested it and it works.